example-2/styles.css

```diff
@@ -0,0 +1,110 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-size: 16px;
+  background: #000000;
+  color: #ffffff;
+}
+
+.wrapper {
+  display: flex;
+  margin: 40px auto 0;
+  width: 80%;
+  max-width: 718px;
+}
+
+input {
+  color: white;
```

Comment:
super nitpick but on line 6 you do #ffffff and here you do white. Would do the hex to be consistent. Sorry about this nitpick